src/EurobusinessHelper.UI.ASP/app/src/Pages/GamePage/CurrentGame/CurrentGame.tsx

```diff
@@ -27,37 +28,57 @@ interface CurrentGameProps {
 export const CurrentGame = ({ gameId }: CurrentGameProps) => {
   const [amount, setAmount] = useState<string>("");
   const [errorMessage, setErrorMessage] = useState<string>("");
+  const [operationLog, setOperationLog] = useState<string[]>([]);
   const [, setHub] = useState<GameHub | undefined>(undefined);
 
   const gameDetails = useSelector(selectGameDetails(gameId));
   const identity = useAppSelector(selectIdentity);
 
   const dispatch = useDispatch();
   const currentAccountId = useMemo(
-    () =>
-      gameDetails?.accounts?.find((a) => a.email === identity?.email)?.id || "",
+    () => gameDetails?.accounts?.find((a) => a.email === identity?.email)?.id || "",
     [gameDetails, identity]
   );
 
+  const getAccountNameAndEmail = useCallback(
+    (accountId: string) => {
+      const account = gameDetails?.accounts?.find((a) => a.id === accountId);
+
+      return `${account?.name} (${account?.email})`;
+    },
+    [gameDetails]
+  );
+
   useEffect(() => {
     if (!currentAccountId) return;
     const hub = new GameHub(
       () => dispatch(fetchDetails(gameId, true)),
       (requestId, accountTo, amount) => {
-        if (
-          window.confirm(
-            `Account ${accountTo} requested $${amount}. Request id: ${requestId}`
-          )
-        )
+        if (window.confirm(`Account ${getAccountNameAndEmail(accountTo)} requested $${amount}.`))
```

Comment:
The messages are not consistent - one is terminated with a period, the other is not.
Also, now the messages are the same for both events - how is the user supposed to know if the money is requested from them or from the bank?